lib/main.coffee

```diff
@@ -29,7 +39,7 @@ module.exports =
 
     # Check version of coala
     version = helpers.exec(@executable, ['--version']).then (result) ->
-      if result <= COALA_MIN_VERSION
+      if cmpVersions(result, COALA_MIN_VERSION) < 0
```

Comment:
Well if there's something similar you could find for js/coffee, thatd be great. Otherwise I guess we'll just use this for now.